tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_jumpf_validation.py

```diff
@@ -391,7 +390,7 @@ def test_jumpf_to_returning_variable_stack_3(eof_test: EOFTestFiller):
                 Section.Code(
                     code=Op.REVERT(0, 0),
                     code_inputs=1,
-                    max_stack_height=3,
```

Comment:
difference here is != code_inputs, but it looks sensible. Can you confirm this is intended?

Reply:
Looks like typo